.github/workflows/e2e.yml

```diff
@@ -137,7 +137,6 @@ jobs:
         # Run tests
         /usr/local/bin/ginkgo --nodes=25                \
           --focus="Netpol"  \
-          --skip="SCTP"  \
```

Comment:
`[Feature:SCTPConnectivity]` is the tests that actually use SCTP packets. There are some other tests with "SCTP" in their name that create policies that reference SCTP but which do not actually test SCTP packets, and we should absolutely be running those ones.

Reply:
we must run all, is just to avoid noise, at this stage , once the containerd bug was sorted out, this and scale testing are the remaining issues